app/routes/campaigns.js

```diff
@@ -32,6 +32,12 @@ function fetchCampaign(campaignDoc) {
       .then((keywords) => {
         campaign.keywords = keywords;
 
+        return contentful.fetchCampaign(campaign.id);
+      })
+      .then((contentfulCampaign) => {
+        campaign.overrides = contentfulCampaign.fields;
+        delete campaign.overrides.campaignId;
```

Comment:
Will look to deprecate `delete` usage once we take on #784 per https://github.com/DoSomething/gambit/pull/783#discussion_r100140281